src/components/directory-content.jsx

```diff
@@ -993,6 +578,7 @@ const DirectoryContent = () => {
                         setActiveElement(null);
                         setOpenDescModificationDialog(false);
                     }}
+                    updateElement={updateElement}
```

Comment:
I don't understand, does that mean it did not work before ?

Reply:
Yup, wasn't supposed to be in the PR and it was fixed it the meantime